clang/include/clang/Sema/Sema.h

```diff
@@ -12303,6 +12303,7 @@ class Sema final {
   };
 
   bool isKnownGoodSYCLDecl(const Decl *D);
+  void CheckVarDeclOKIfInKernel(VarDecl *var);
```

Comment:
You likely cannot use this name, as this would imply it also affects CUDA, OMP, and OpenCL.

Reply:
Also, where is this defined?  I don't see it in this review.

Reply:
Should I put "SYCL" into its name?

Reply:
I have rename this to be `checkSYCLVarDeclIfInKernel`  and the variable param is capitalized.  Let me know if there is anything else.